pool-interface/src/reserve_config.rs

```diff
@@ -115,3 +126,15 @@ pub struct InitReserveInput {
 fn zero_bytes<const N: usize>(env: &Env) -> BytesN<N> {
     BytesN::from_array(env, &[0; N])
 }
+
+///Collateralization parameters
+#[contracttype]
+#[derive(Clone, Copy)]
+pub struct CollateralParams {
```

Comment:
Suggest to name custom types that used as input and don't used into inner logic with `Input` postfix, like `InitReserveInput`